### PR TITLE
Add Lyapunov workflow examples

### PR DESCRIPTION
**Pull request by jaewookjsuh**
- `examples_lyapunov/`: add Lyapunov workflow example notebooks
  - FEG / Dual-FEG
  - OC-Halpern / Dual-OC-Halpern
  - Dual-OHM
  - OGM / OGM without z / OGM-G
  - BPGM
  - PPM for monotone operators
- `lyapunov_utils.py`: skip infeasible basis candidates that cannot represent `V`
- `ipython_utils.py`: handle empty `(0, 0)` matrices gracefully in `pprint_labeled_matrix`
  - `ipython_utils_test.py`: add regression test for empty matrix display handling
- `pyproject.toml`: add explicit setuptools package discovery config for `pepflow`

### CODE REVIEW

File: pepflow/lyapunov_utils.py [hunk 1]
```diff
@@ -402,12 +402,16 @@ def find_basis_with_sparsest_coefficients(
...
-            span_tol=span_tol,
-        )
+        try:
+            coeff_matrix = _find_symmetric_coefficient_matrix_from_coords(
+                V_coords,
+                candidate_basis_coords,
+                indep_tol=indep_tol,
+                span_tol=span_tol,
+            )
+        except ValueError:
```
Comment (BichengYing) [inline pepflow/lyapunov_utils.py]: ValueError is too generic so that we may capture the real exception.  Try a customized exception

**Review by BichengYing — APPROVED** [review]
I didn't carefully check the example ipynb. Randomly pick one and scan. It seems fine. Let's check into it for the arxiv paper reference

File: pepflow/lyapunov_utils.py (hunk 1 above)
Comment (jaewookjsuh) [reply pepflow/lyapunov_utils.py]: I understand. I will address it. Thank you for your suggestion.
